### ISSUE TITLE
A question about lr_decay_epoch

### ISSUE THREAD

**Issue by SakurajimaMaiii** [dnap512/SelfReg#6]
Hi, thanks for your great work and code release.
I have a question about training.

It is noticed that in _train.ipynb_       
**lr_decay-epoch is set to 100, but max epoch is 30.**
**In paper, 'note that such a decaying learning rate is not used when ti combined with the Stochastic weights averaging technique'**

It means that: if we use SWA(stochastic weights averaging), we use constant lr (0.004) during training.
                     if we not use SWA,  we need to decay lr to 0.004*0.1=0.0004 after epoch 24.
Am I right ?
Another question is :
the results of these 2 settings differ a lot ?
Thanks for your help.

**Comment by dnap512** [comment]
Hi, @SakurajimaMaiii !

> It means that: if we use SWA(stochastic weights averaging), we use constant lr (0.004) during training.

Yes, you're right.
If you don't want to use SWA, you should set lr_decay_epoch to 24 and modify some codes about SWA (i.e. swa_model, best_model_wts ... in utils.py) properly.

As I remember, w/ SWA setting is higher about 0.3% on average.

Thank you for your interest in our research.

**Comment by SakurajimaMaiii** [comment]
Hi, I have another problem.
In single source domain experiment,
the hyperparameters are same ?
i.e.  learning rate, batch size , epoch. 
Thanks for your help.

**Comment by dnap512** [comment]
Yep @SakurajimaMaiii , All hyperparameters are the same.

**Comment by SakurajimaMaiii** [comment]
> Yep @SakurajimaMaiii , All hyperparameters are the same.

Wai,  so quick . Thanks for your reply.
